tasks/main.yml

```diff
@@ -12,7 +12,7 @@
 
 - name: Download helm.
   unarchive:
-    src: https://get.helm.sh/helm-{{ helm_version }}-{{ helm_platform }}-{{ helm_arch }}.tar.gz
+    src: "{{ helm_repo_path }}helm-{{ helm_version }}-{{ helm_platform }}-{{ helm_arch }}.tar.gz"
```

Comment:
You’re missing a ‘/‘ after the ‘“ {{ helm_repo_path }}’ as is this will break the module.

Reply:
Thank's, I fixed it.